tests/bpf2c_tests/elf_bpf.cpp

```diff
@@ -11,6 +11,7 @@
 #include "catch_wrapper.hpp"
 
 #define main test_main
+#define ENABLE_SKIP_VERIFY
```

Comment:
Can we enable this by default for DEBUG builds? Similar was done for interpreter mode. The default behavior for bpf2c.exe would still remain the same for DEBUG builds also.

Allowing to skip verification (though only in debug) will help in following case:
It will allow to generate C code from an invalid ELF file and loading such a program will exercise the negative scenarios being handled in ebpfcore.

Reply:
Done.